src/Raven.Studio/typescript/components/pages/database/settings/customAnalyzers/DatabaseCustomAnalyzers.tsx

```diff
@@ -77,156 +79,186 @@ export default function DatabaseCustomAnalyzers({ db }: NonShardedViewProps) {
     const isLimitReached = databaseLimitReachStatus === "limitReached" || clusterLimitReachStatus === "limitReached";
 
     return (
-        <div className="content-margin">
-            <Col xxl={12}>
-                <Row className="gy-sm">
-                    <Col>
-                        <AboutViewHeading title="Custom analyzers" icon="custom-analyzers" />
-                        {isDatabaseAdmin && (
-                            <>
-                                <div id="newCustomAnalyzer" className="w-fit-content">
+        <>
+            {clusterLimitReachStatus !== "notReached" && (
+                <Alert
+                    color={clusterLimitReachStatus === "limitReached" ? "danger" : "warning"}
+                    className="text-center mb-3"
+                >
+                    <Icon icon="cluster" />
+                    Cluster {clusterLimitReachStatus === "limitReached" ? "has reached" : "is reaching"} the{" "}
+                    <strong>maximum number of Custom Analyzers</strong> allowed per cluster by your license{" "}
+                    <strong>
+                        ({numberOfCustomAnalyzersInCluster}/{licenseClusterLimit})
+                    </strong>
+                    <br /> Delete unused analyzers or{" "}
+                    <strong>
+                        <a href={upgradeLicenseLink} target="_blank">
+                            upgrade your license
+                        </a>
+                    </strong>
+                </Alert>
+            )}
```

Comment:
Should there also be a database limit alert? Or we leave a badge?

Reply:
I've added the database limit alert to make it consistent with other views